roles/mysql_8_4/defaults/main.yml

```diff
@@ -11,11 +12,12 @@ mysql_config_file: /etc/mysql/my.cnf
 mysql_8_4_socket: /var/run/mysqld/mysqld.sock
 
 MYSQL_8_4_APT_KEYSERVER: keyserver.ubuntu.com
-MYSQL_8_4_APT_KEY: A8D3785C
+# FROM: https://repo.mysql.com/apt/ubuntu/conf/distributions
+MYSQL_8_4_APT_KEY: B7B3B788A8D3785C
 MYSQL_8_4_REPO: deb http://repo.mysql.com/apt//ubuntu/ {{ ansible_distribution_release }} mysql-8.4-lts
 
 MYSQL_8_4_SUPPORTED_DISTRIBUTIONS:
-  - jammy
+  - noble
```

Comment:
Is it still possible to install MySQL in Jammy after these changes? Will only Noble be available?

Reply:
The thing is that if we still support Jammy as well, we'll need to test it in molecule